codecov.yml

```diff
@@ -1,3 +1,3 @@
 ignore:
-  - "tests/utils/testutils.py"
-  - "tests/utils/testutils_lc.py"
+  - "tests/utils/"
+
```

Comment:
For the future, can we explore [commenting](https://docs.codecov.com/docs/codecovyml-reference#comment)?